index.html

```diff
@@ -0,0 +1,293 @@
+            <li><a class="link partner-link" href="#sponsor">Sponsor1</a></li>    
+        </ul>
+    </div>
+    
+    <main class="main" id="home">
+    
+        <nav class="nav">
+            <ul class="nav-top-list">
+                <li><a href="https://www.facebook.com/oybek.kayumov.39/" target="_blank" rel="noopener noreferrer"><i class="fab fa-facebook fa-lg"></i>
+                    </a></li>
+                <li><a href="https://twitter.com/KayumovOybek" target="_blank" rel="noopener noreferrer">
+                        <i class="fab fa-twitter fa-lg"></i>
+                    </a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+            <div class="nav-bottom">
+                <a class="logo" href="#home"> <img alt="logo" src="images/logo.png" height="90" width="180">
+                </a>
+                <ul class="nav-bottom-list">
+                    <li><a class="link about-link" href="#about">About</a></li>
+                    <li><a class="link" href="#program">Program</a></li>
+                    <li><a class="link" href="#join">Join</a></li>
+                    <li><a class="link" href="#partner">Sponsor</a></li>
+                    <li><a class="link" href="#news">News</a></li>
+                    <li class="nav-list-button"><a class="link" href="#home">CC Campaign</a></li>
+                </ul>
+            </div>
+        </nav>
+    
+        <header class="head">
+            <i class="fas fa-bars fa-3x hamburger open"></i>
+            <h1 class="head-top">"21st International Exhibition"</h1>
+            <h2 class="head-main">Food, Ingredients and Food Technologies</h2>
+            <p class="head-p">
+                UzFood is the key event in the food industry. Every year the exhibition is attended by leading producers, suppliers and distributors of the food industry representing both national and global food markets. <br> The exhibition is annually visited by top managers of the leading companies, importers and distributors.
+            </p>
+            <h2 class="head-data">2022.03.29(TUE) ~ 31(THU)</h2>
+            <h2 class="head-place">@Uzexpocentre, Tashkent, Uzbekistan</h2>    
+        </header>
+    
+        <section class="program">
+            <h2 class="program-title">Main Program</h2>
+            <div class="red-line"></div>
+            <div class="cards">
+                <div class="card">
+                    <div class="lecture">
+                        <img class="lecture-icon" src="images/icons/lecturer.png" alt="lecture">
+                        <h2>Lecture</h2>
+                    </div>
+                    <p class="card-p">
+                        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nesciunt sint praesentium iste possimus
+                        numquam. Aperiam!
+                    </p>
+                </div>
```

Comment:
- [ ] To match the original design better, you should consider adding a `background-color` to your cards.

Original design:
![image](https://user-images.githubusercontent.com/66967274/147248863-57ffbc97-58ab-4771-a1ca-0e5c3ae567dc.png)


Your design:
![image](https://user-images.githubusercontent.com/66967274/147248841-bdb6e0e2-37fc-4057-8cfa-7d578ab3e803.png)

Reply:
updated